sys/include/net/gnrc/tcp/config.h

```diff
@@ -7,6 +7,8 @@
  */
 
 /**
+ * @defgroup    net_gnrc_tcp_conf   GNRC TCP compile configurations
```

Comment:
Somehow Murdock does not like this :-/

Reply:
Why is Travis passing the static check? :confused: